components/Logo/logo.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { Story, Meta } from "@storybook/react/types-6-0";
+import Logo, { LogoProps } from "./logo";
+
+export default {
+  title: "Common/Logo",
+  component: Logo,
+} as Meta;
+
+const Template: Story<LogoProps> = (args) => <Logo {...args} />;
+
+export const TextLogo = Template.bind({});
+TextLogo.args = {
+  src: "/text-logo.svg",
+};
+
+export const FahrerLogo = Template.bind({});
+FahrerLogo.args = {
```

Comment:
Don`t switch between english and german ;-)

```suggestion
export const DriverLogo = Template.bind({});
DriverLogo.args = {
```